WINDOWS.md

```diff
@@ -0,0 +1,165 @@
+# Setup Windows
+
+## Table of Contents
+
+1. [Setup Docker](#setup-docker)
+
+2. [Setup the application](#Setup-the-application)
+
+3. [Run the application](#Run-the-application)
+
+4. [Using the application](#using-the-application)
+
+5. [Common setup errors](#having-troubles-check-our-common-errors--gotchas)
+
+6. [Development](#development)
+
+7. [Backup and restore the Terrastories database](#backup-and-restore-the-terrastories-database)
+
+## Setup Docker
+
+Docker is a platform that serves to deliver software in packages called containers. Terrastories makes use of this to deliver the 
+many facets included in the application in one easy package. 
+
+Download Docker:
+
+- https://hub.docker.com/editions/community/docker-ce-desktop-windows/
+
+System prerequisites for Windows 10 Pro: 
+- 64 bit processor 
+- 4GB system RAM 
+- [virtualization must be enabled](https://mashtips.com/enable-virtualization-windows-10/). 
+
+Setup instructions for Docker on Windows 10 Pro:
+
+- https://docs.docker.com/docker-for-windows/install/
+
+System prerequisites for Windows 10 Home: 
+- Ensure that you are running on version 2004 or higher, you will need to update your system if it is running on an older version
+- Enable the WSL 2 feature on Windows, follow the instructions on this link to complete this step, this includes installing a Linux distribution such as Ubuntu
+- To complete the previous steps, you must have a 64 bit processor, 4GB system RAM, and [virtualization must be enabled](https://mashtips.com/enable-virtualization-windows-10/). 
+
+Setup instructions for Docker on Windows 10 Home:
+
+- https://docs.docker.com/docker-for-windows/install-windows-home/
+
+There is no need to import any repositories at this point through docker. 
+To verify that your docker is up and running, click on the app or check the menu bar. Docker should show that it is running: 
+<br>
+<div style="text-align:center">
+    <img src="dockerMac.png" alt="dockerImg" width="200"/>
+</div>
```

Comment:
Did you mean to use `dockerMac.png` image in the windows setup? I wonder if we should screenshot for windows, or leave out this image entirely on this section for windows.